app/graphql/types/query_type.rb

```diff
@@ -1,5 +1,6 @@
 module Types
   class QueryType < Types::BaseObject
+    field :all_recipes, resolver: Queries::Recipes::AllRecipes
```

Comment:
I think we should keep queries with their resolvers all in this file. if you look at the get_user query under this comment or my open pr it formats queries differently than mutations. I like this way of doing it because usually were not doing much logic within a query resolver